ISISScientificComputing/autoreduce
# Adds AR workspace. Ignores developer-specific settings.json

This is based on #1042 and should be merged after

### Summary of work
Adds the common settings into an AR VSCode workspace. This allows for developer specific settings inside `.vscode/settings.json` (such as python exe path), while enforcing common workspace settings.

### How to test your work
With VSCode open the `autoreduce.code-workspace` as a workspace:

` File > Open Workspace > Select autoreduce.code-workspace `

Extensions should be recommended, if you don't have them already.

Fixes #1175
